packages/ui/src/form/DynamicFormField/DynamicInput.tsx

```diff
@@ -29,11 +32,14 @@ const DynamicInput: FunctionComponent<DynamicInputProps> = ({
     onChange = noop,
     options,
     placeholder,
+    useFloatingLabel,
     value,
     ...rest
 }) => {
-    const { date } = useLocale();
-    const { inputFormat } = date || { inputFormat: '' };
+    // FIXME: useLocale must be used within a LocaleContextProvider
```

Comment:
Are we planning to make the fix as part of this PR?

Reply:
`locale` is not working right now, @bc-peng told me that there are several tickets pending to make it work.